tracing-subscriber/src/fmt/mod.rs

```diff
@@ -18,6 +18,12 @@ mod span;
 pub mod time;
 pub mod writer;
 
+#[cfg(feature = "env-logger-compat")]
+mod env_logger_compat;
+#[cfg(feature = "env-logger-compat")]
```

Comment:
I'd prefer not to introduce a new feature for this. Can we drop the "env_logger_compat" name?